rinja/src/helpers.rs

```diff
@@ -4,6 +4,8 @@ use std::cell::Cell;
 use std::fmt;
 use std::iter::{Enumerate, Peekable};
 
+pub use {core, std};
```

Comment:
Please still add a comment explaining why we have this import.

Reply:
Also: the `helpers` module is also public and therefore might be imported as `use rinja::helpers::*`. So maybe put it into a hidden `reexport` module or something along the line?

Reply:
It is `#[doc(hidden)]`. If someone does indeed glob-imports it, then they deserve all the pain they get.

Reply:
Hum no it's public: https://github.com/rinja-rs/rinja/blob/master/rinja/src/lib.rs#L64. Or maybe you meant something else? But currently as I understand it, the module is public and the reexports are public too.

Reply:
I placed the attribute inside the module itself:
<https://github.com/rinja-rs/rinja/blob/4b4418980909122bcbd187f093934a8a699af3e2/rinja/src/helpers.rs#L1>

Maybe that was not the best idea. :)

Reply:
Damn, completely missed it, sorry about that. If you don't mind, please put it on the import instead so future me won't be lost. :laughing:

Reply:
Hehe, yeah, it was a quite unconventional placement. :)